pkg/manifests/parser/parser.go

```diff
@@ -92,6 +95,38 @@ func resourceInfoToK8sObject(info *resource.Info, l logger.Logger, muteErrsAndWa
 	return &resObject, nil
 }
 
+func CreatePodK8sObject(pod *v1.Pod) K8sObject {
```

Comment:
It would be cumbersome.
First, I want to create `K8sObject`s before creating `policyEngine`. For example, in `connlist` I need those objects for `connsListFromParsedResources` function, which further creates the engine with `NewPolicyEngineWithObjects` function.
Second, even if I created an engine without objects first, I would have to use the deprecated function `SetResources` to create `K8sObject`s.

Reply:
Done